teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(age, specialization)
+    super(age)
+    @specialization = specialization
+  end
```

Comment:
- [ ] [REQUIRED] Good job so far! I think you missed `name` and `parent_permission`, your `def initialize` should like that:

```ruby
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```